src/koans/01_equalities.clj

```diff
@@ -32,5 +32,8 @@
   "Symbolism is all around us"
   (= 'hello (symbol __))
 
+  "Introducing nil"
```

Comment:
Yes, sorry I didn't give much thoughts to it as English is not my first language... If anybody as any suggestion...